packages/core/src/Input.js

```diff
@@ -17,7 +17,9 @@ const borders = ({ color, theme }) => {
   }
 }
 
-const Input = styled.input`
+const Input = styled.input.attrs({
```

Comment:
If this is just a `styled.input` shouldn't it be able to receive readonly inherently, without needing a readonly prop to be defined explicitly?